core/domain/subscription_services_test.py

```diff
@@ -36,6 +36,8 @@
 FEEDBACK_THREAD_ID_2 = 'fthread_id_2'
 USER_ID = 'user_id'
 
+# Remove this line, maybe by shortaning names of methods smaller
+# pylint: disable=C0301
```

Comment:
use the 'word' form of the warning, rather than the numeric code -- the former is more understandable. (You can search for "pylint: disable" in the codebase to get an idea of how this works.)